scripts/build_dev_server_docker

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o xtrace
+
+readonly CACHE_DIR='bazel-cache'
+readonly SCRIPTS_DIR="$(dirname "$0")"
+
+"$SCRIPTS_DIR/docker_run" bazel \
+    --output_base="$CACHE_DIR/dev" \
+    build \
+    --symlink_prefix='dev-' \
```

Comment:
Why do we need a separate prefix? I think the default workspace-level one should be fine?

Reply:
I think this is needed. Otherwise, this will always invalidate the cache of the proper server as it changes the compiler.